sdk/java/core/src/test/kotlin/com/keepersecurity/secretsManager/core/SecretsManagerTest.kt

```diff
@@ -130,6 +130,7 @@ internal class SecretsManagerTest {
 //            record.updatePassword("new password")
 //            updateSecret(options, record)
         }
+//        deleteSecret(options, listOf("7LMCSfr3D1Z04Nh3LsGqYQ"))
```

Comment:
Since this repo is public, we are trying to to put any “internal dirt” into it. Undo this file.

Reply:
understood.  It's been undone.  Thanks!